Type error; missing "declare" modifier

Running into a type error when trying to build TypeScript projects. Not quite sure if it's due to a feature in newer version of TS or a TS option, however this is the resulting error:

```sh
$ tsc --noEmit --allowJs --checkJs
node_modules/rollup-plugin-svelte/index.d.ts:10:1 - error TS1046: Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.

10 class CssWriter {
   ~~~~~


Found 1 error.

error Command failed with exit code 1.
```

### Versions

- `typescript`: `3.9.7`
- `rollup-plugin-svelte`: `5.2.3`

--- a/index.d.ts
+++ b/index.d.ts
@@ -7,7 +7,7 @@ interface Css {
   map: any;
 }
 
-class CssWriter {
+declare class CssWriter {
   code: string;
   map: {
     version: number;

